go/libraries/doltcore/sqle/table_editor_fk_test.go

```diff
@@ -384,7 +379,9 @@ func assertTableEditorRows(t *testing.T, root *doltdb.RootValue, expected []sql.
 			sqlRows = append(sqlRows, sqlRow)
 			return nil
 		})
-		assert.Equal(t, convertSqlRowToInt64(expected), sqlRows)
+		if !assert.Equal(t, convertSqlRowToInt64(expected), sqlRows) {
+			fmt.Println("")
```

Comment:
I'm assuming you wrote something here and forgot to remove it afterward lol

Reply:
Yeah I needed somewhere to put a breakpoint, good catch